Tweeter.Tests/DAL/TweeterRepoTests.cs

```diff
@@ -110,5 +135,51 @@ public void RepoEnsureUsernameExistsOfTwit()
             // Assert
             Assert.IsNotNull(found_twit);
         }
+
+        [TestMethod]
+        public void CanGetAllTweets()
+        {
+            //Arrange
+            ConnectToDatastore();
+         
+            //Act    
+            List<Tweet> tweets = Repo.GetTweets();
+            int expected = 2;
+            int actual = tweets.Count();
+            
+            //Assert 
+            Assert.AreEqual(expected, actual);
+        }
+
+        [TestMethod]
+        public void CanAddTweet()
+        {
+            //Arrange 
+            ConnectToDatastore();
+
+            //Act
+            Tweet bogus_tweet = new Tweet { TweetId = 3, Message = "Hey homes." };
+            Repo.AddTweet(bogus_tweet);
+            List<Tweet> tweets = Repo.GetTweets();
+
+            //Assert
+            Assert.AreEqual(3, tweets.Count());
```

Comment:
Arrange->Act->Assert is a nice framework to write tests in, but sometimes it's nice to have multiple asserts at different points in your test. For example, it'd be nice in this test to have an `Assert.AreEqual(2, tweets.Count());` before the Act portion, just to be sure there are for sure only 2 tweets in the test db.